packages/python-solv/python-solv.spec

```diff
@@ -0,0 +1,150 @@
+BuildRequires:  python%{python3_pkgversion}-devel
+
+# -DWITH_LIBXML2=ON
+BuildRequires:  libxml2-devel
+
+# -DWITH_OPENSSL=ON
+BuildRequires:  pkgconfig(openssl)
+
+# -DENABLE_LZMA_COMPRESSION=ON
+BuildRequires:  xz-devel
+
+# -DENABLE_BZIP2_COMPRESSION=ON
+BuildRequires:  bzip2-devel
+
+%if %{with zstd}
+# -DENABLE_ZSTD_COMPRESSION=ON
+BuildRequires:  libzstd-devel
+%endif
+
+%if %{with zchunk}
+# -DENABLE_ZCHUNK_COMPRESSION=ON
+BuildRequires:  libzck-devel
+%endif
+
+%description
+Python bindings for the %{name} library.
+
+Python 3 version.
+
+
+
+%prep
+%autosetup -p1 -n lib%{libname}-%{version}
+
+%build
+%cmake -GNinja                                            \
+  -DFEDORA=1                                              \
+  -DENABLE_RPMDB=ON                                       \
+  -DENABLE_RPMDB_BYRPMHEADER=ON                           \
+  -DENABLE_RPMDB_LIBRPM=ON                                \
+  -DENABLE_RPMPKG_LIBRPM=ON                               \
+  -DENABLE_RPMMD=ON                                       \
+  -DENABLE_COMPS=%{__cmake_switch -b comps}               \
+  -DENABLE_APPDATA=%{__cmake_switch -b appdata}           \
+  -DUSE_VENDORDIRS=ON                                     \
+  -DWITH_LIBXML2=ON                                       \
+  -DWITH_OPENSSL=ON                                       \
+  -DENABLE_LZMA_COMPRESSION=ON                            \
+  -DENABLE_BZIP2_COMPRESSION=ON                           \
+  -DENABLE_ZSTD_COMPRESSION=%{__cmake_switch -b zstd}     \
+  -DENABLE_ZCHUNK_COMPRESSION=%{__cmake_switch -b zchunk} \
+%if %{with zchunk}
+  -DWITH_SYSTEM_ZCHUNK=ON                                 \
+%endif
+  -DENABLE_HELIXREPO=%{__cmake_switch -b helix_repo}      \
+  -DENABLE_SUSEREPO=%{__cmake_switch -b suse_repo}        \
+  -DENABLE_DEBIAN=%{__cmake_switch -b debian_repo}        \
+  -DENABLE_ARCHREPO=%{__cmake_switch -b arch_repo}        \
+  -DMULTI_SEMANTICS=%{__cmake_switch -b multi_semantics}  \
+  -DENABLE_COMPLEX_DEPS=%{__cmake_switch -b complex_deps} \
+  -DENABLE_CONDA=%{__cmake_switch -b conda}               \
+  -DENABLE_PYTHON=%{__cmake_switch -b python_bindings}    \
+  -DENABLE_STATIC=ON                                      \
+  -DDISABLE_SHARED=ON                                     \
+  -DPYTHON_LIBRARY=/usr/lib64/libpython3.11.so.1.0        \
+  -DPYTHON_INCLUDE_DIR=/usr/include/python3.11            \
+  -DPYTHON_EXECUTABLE=%{python3}                          \
+  %{nil}
+%cmake_build
+
+%install
+%cmake_install
+
+
+rm -rf %{buildroot}/usr/bin/
+rm -rf %{buildroot}/usr/include/
+rm -rf %{buildroot}%{_libdir}/debug/
+rm -rf %{buildroot}/usr/bin/
+rm -rf %{buildroot}%{_datadir}
+
+rm %{buildroot}/usr/lib64/libsolv*.a
+rm %{buildroot}/usr/lib64/pkgconfig/libsolv*.pc
+
+
+%check
+%ctest
+
+# Python smoke test (not tested in %%ctest):
+export PYTHONPATH=%{buildroot}%{python3_sitearch}
+export LD_LIBRARY_PATH=%{buildroot}%{_libdir}
+%python3 -c 'import solv'
+
+%files
```

Comment:
I'd argue the resulting package binary should be called `python3.11-solv`, not `python-solv`.

Reply:
+1